contrib/nvenc/module.defs

```diff
@@ -0,0 +1,21 @@
+$(eval $(call import.MODULE.defs,NVENC,nvenc))
+$(eval $(call import.CONTRIB.defs,NVENC))
+
+NVENC.FETCH.url      = https://download.handbrake.fr/contrib/nv-codec-headers-8.1.24.2.tar.gz
+NVENC.FETCH.url     += https://github.com/FFmpeg/nv-codec-headers/releases/download/n8.1.24.2/nv-codec-headers-8.1.24.2.tar.gz
+NVENC.FETCH.sha256   = 2b339b2493f5b63b285dd660be05102af7d4ff2de9fcf3af94e48b5f91a3ca57
+NVENC.FETCH.basename = nv-codec-headers-n8.1.24.2.tar.gz
```

Comment:
Not sure this is correct/necessary.

Reply:
Notice the "n" before the 8 so I believe you have to set the basename for that when the internal folder isn't matching the tar.

Reply:
Yes, I see that now. I think the .tar.gz needs removed though.

Reply:
`NVENC.EXTRACT.tarbase = nv-codec-headers-n8.1.24.2`